README.md

```diff
@@ -1,3 +1,12 @@
 # backend-sdk-testing
 
 Common test suit for all backend SDKs
+
+# TODO
+
+-   [ ] check for FIXME on api-mock folder
+-   [ ] not sure why python didn't worked with my default docker startup, meanwhile node did
+-   [ ] how to handle mock-server shutdown when tests fail
+-   [ ] mocha .only isn't working when not specifying the test file
+-   [ ] move api-mock-server to their own project
+-   [ ] migrate accountlinking, mfa, accountlinking-with-session tests
```

Comment:
Make a PR from base -> main, and then add this list in there, instead of the readme.

Reply:
ok